CHANGES.md

```diff
@@ -13,6 +13,57 @@ development source code and as such may not be routinely kept up to date.
 
 # __NEXT__
 
+## Improvements
+
+* Commands that use a runtime (`nextstrain build`, `nextstrain shell`, and
+  `nextstrain view`) now support two new options for setting or passing thru
+  environment variables into the runtime environment:
+
+      --env <name>[=<value>]
+      --envdir <path>
+
+  When either of these options are given, the default behaviour of
+  automatically passing thru several "well-known" environment variables is
+  disabled.  That is, the following "well-known" environment variables are only
+  automatically passed thru when the new `--env` and `--envdir` options are
+  _not_ used:
```

Comment:
> One thing I want to consider (and think we should do) to help deprecate and remove the automatic hostenv forwarding is to disable it when these new `--env` or `--envdir` options are used. Since they're new, their use is a good time/indicator to push folks to spell out what they need without hostenv. Thoughts?

Makes sense to me!